src/pages/Suitcase/SuitcaseStyles.tsx

```diff
@@ -0,0 +1,62 @@
+import { createStyles, Theme } from '@material-ui/core/styles';
+
+export const styles = (theme: Theme) => 
+    createStyles({
+      card: {
+        display: 'flex',
+        width: '100%',
+        borderRadius: '20px 20px 0px 0px',
+        height: '35%',
+        backgroundColor: theme.palette.white.main, 
+        boxShadow:
+          '0px 14px 50px 20px rgba(0,0,0,0.2)',
+        position: 'absolute', 
+        bottom: '10%', 
+      },
+      img: {
+        objectFit: 'contain', 
+        width: '30%', 
+        margin: '2%'
+      },
+      rightSide: {
+        display: 'flex',
+        flexDirection: 'column', 
+        margin: '3.5rem 1.0rem 2.0rem 1.0rem',
+      },
+      cancel: {
+        fontSize: 35, 
+        padding: '1.0rem', 
+        right: '1%',
+        position: 'absolute'
+      },
+      buttonsRow: {
+        display: 'flex',
+        flexDirection: 'row', 
+        justifyContent: 'space-around', 
+        margin: '2.0rem', 
+        fontSize: '12px',
+        fontWeight: 600, 
+      },
+      manualsButton: {
+        fontSize: 'inherit',
```

Comment:
re: disc about styling -- I think `fontSize: 'inherit', fontWeight: 'inherit',` might be the cause of your mention of the Link component overriding your styling -- 

one recommendation would be to remove these two from `manualsButton` and `helpButton` and instead use <Typography> inside AnimationCard.tsx!

Reply:
Hmm I believe it styled the text, but not the flex aspect. I wonder if we'll really need flex adjustments tho bc this mobile not web? 
![Screen Shot 2021-04-16 at 10 02 48 PM](https://user-images.githubusercontent.com/63089231/115102414-7dd6a480-9eff-11eb-9de1-bc17797e28b3.png)

Reply:
Oooh sorry just saw this comment, hmm I'll take a look !!